bors.toml

```
@@ -2,9 +2,9 @@ status = [
   "Julia 1.% - ubuntu-latest - x64",
   "Julia 1.% - macOS-latest - x64",
   "Julia 1.% - windows-latest - x64",
-  "Julia 1 - ubuntu-latest - x64",
-  "Julia 1 - macOS-latest - x64",
-  "Julia 1 - windows-latest - x64",
+  "Julia 1.6 - ubuntu-latest - x64",
+  "Julia 1.6 - macOS-latest - x64",
+  "Julia 1.6 - windows-latest - x64",
```

This change seems wrong, that case is already covered above. Changing this means that it will pass regardless of whether tests fail with the latest stable version of Julia or not.
```suggestion
  "Julia 1 - ubuntu-latest - x64",
  "Julia 1 - macOS-latest - x64",
  "Julia 1 - windows-latest - x64",
```